fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -61,7 +61,7 @@ object FaciaContent {
 
     CuratedContent(
       content,
-      trailMetaData.headline.orElse(contentFields.get("headline")),
```

Comment:
Are we sure we don't need to favour `contentFields.headline` over `content.webTitle`?